### ISSUE TITLE
🐛 Bug Report: Pingvin crashes after user puts wrong credentials and requests password reset

### ISSUE THREAD

**Issue by sassanix** [stonith404/pingvin-share#420]
### 👟 Reproduction steps

When a user puts in the wrong password multiple times, and then tries to reset their password. The whole pingvin-share crashes.

### 👍 Expected behavior

To send a password reset link by email.

### 👎 Actual Behavior

Crashed and had to manually restart the pingvin share.

### 🌐 Browser

Safari

**Comment by stonith404** [comment]
Can you share the logs?

**Comment by sassanix** [comment]
> Can you share the logs?

Where can I find it?

**Comment by stonith404** [comment]
Thanks and sorry for my late reply. Yeah it looks like the logs got cleared. Would you mind to reproduce the bug again and check the logs again? The logs shouldn't get cleared if you restart the container with `docker compose restart`.

**Comment by sassanix** [comment]
> Thanks and sorry for my late reply. Yeah it looks like the logs got cleared. Would you mind to reproduce the bug again and check the logs again? The logs shouldn't get cleared if you restart the container with `docker compose restart`.

I will try to recreate it.

**Comment by stonith404** [comment]
Any updated on this?

**Comment by sassanix** [comment]
I recreated the issue.

I put my password in the wrong multiple times. Then I clicked on the wrong password, and the whole pingvine crashed.

Here are the logs: 

```
/opt/app/backend/dist/src/auth/auth.service.js:81
            throw new common_1.BadRequestException("User not found");
                  ^

BadRequestException: User not found
    at AuthService.requestResetPassword (/opt/app/backend/dist/src/auth/auth.service.js:81:19) {
  response: { message: 'User not found', error: 'Bad Request', statusCode: 400 },
  status: 400,
  options: {}
}

/opt/app/backend/dist/src/auth/auth.service.js:81
            throw new common_1.BadRequestException("User not found");
                  ^

BadRequestException: User not found
    at AuthService.requestResetPassword (/opt/app/backend/dist/src/auth/auth.service.js:81:19) {
  response: { message: 'User not found', error: 'Bad Request', statusCode: 400 },
  status: 400,
  options: {}
}

Node.js v20.11.1
Error [TypeError]: fetch failed
    at context.fetch (file:///opt/app/frontend/node_modules/next/dist/server/web/sandbox/context.js:260:38)
    at Object.middleware [as handler] (file:///opt/app/frontend/.next/server/src/middleware.js:5066:33)
    at <unknown> (file:///opt/app/frontend/.next/server/src/middleware.js:1957:23)
    at AsyncLocalStorage.run (node:async_hooks:346:14)
    at Object.wrap (file:///opt/app/frontend/.next/server/src/middleware.js:1758:24)
    at adapter (file:///opt/app/frontend/.next/server/src/middleware.js:1944:53)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by stonith404** [comment]
Thanks, this is fixed in `v0.23.1`.

**Comment by sassanix** [comment]
Thank you so much!
